submodules/tags/README.md

```diff
@@ -0,0 +1,60 @@
+# Tags
+
+Module to create tags
+
+<!-- markdownlint-disable MD013 MD033 -->
+## Input Variables
+
+| Name            | Type                                                                        | Default Value | Example                                                                               |
+| --------------- | --------------------------------------------------------------------------- | ------------- | ------------------------------------------------------------------------------------- |
+| user_tags       | object({ <br />team = string<br /> purpose = string<br /> owner = string<br /> }) |               | {<br />  team= "devops"<br /> purpose= "elk module test"<br /> owner= "Samir"<br /> } |
+| octopus_tags    | object({ <br />project = string<br /> space = string<br />  })                    |               | {<br />  project = "actions-test"<br /> space   = "Default"<br /> }                   |
+| name            | string                                                                      |               | "Test"                                                                                |
+| octopus_project | string                                                                      |               | "actions-test"                                                                        |
+| octopus_space   | string                                                                      |               | "Default"                                                                             |
```

Comment:
Why do you need this input when you are collecting project and space info in octopus_tags. I don't see these variables either in your variables.tf

Reply:
Done